### PR TITLE
Setup technical documentation page

### PR DESCRIPTION
**Pull request by M-AminAlizadeh**
Here I built technical documentation for javascript language

### CODE REVIEW

File: README.md
Comment (M-AminAlizadeh) [inline README.md:1]: Add your working process here so other people can use and understand it
For example, you can use this [link](https://github.com/microverseinc/readme-template/blob/master/README.md) and try to edit it to your own version

**Review by M-AminAlizadeh — COMMENTED** [review]
Hi @M-AminAlizadeh 
You did a good job as your first pull request 👍



But there are some issues that I listed below:

- _check the comments under the review_
- Didn't use Linters (for more information check this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) -- linters will prevent lots of errors and bugs
- Not documenting your workflow -- It's good to document your workflow to other developers can understand your effort
Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: README.md
Comment (M-AminAlizadeh) [reply README.md:1]: Ok
Thank you for your suggestion

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @M-AminAlizadeh   , 

Great job for implemented  the some of the project requirement  , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ Good Job for Summarize the requested changes.  
✔️ You used correct Git Hub-Flow. 



almost

## Required Changes ♻️
_Check the comments below the review._

- [x] As stated in the project requirement Add as many comments as possible in the given time and finish your review. **Remember to add a minimum of 2 comments** - you will definitely find something that could have been done better!
- [ ] Kindly I recommend you used the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) at least once to leave review comments. Please Make sure the student left the comment under a line or block of code that fits the context the comment is about.
- [x] Kindly make sure that phrased the comments politely, as suggestions instead of orders and always includes an explanation, a "why", for each requested change.
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by M-AminAlizadeh** [comment]
@Abenezer-Tilahun  thank you for your suggestion

**Comment by joskalenda** [comment]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @M-AminAlizadeh , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Great use of github inline feature
✅ ... Good communication with the student

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**